nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/pom.xml

```diff
@@ -77,6 +76,19 @@
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-record-serialization-service-api</artifactId>
```

Comment:
`nifi-record-serialization-service-api` dependency is no longer required. Please remove it.